zfs/ZfsCompressedArcMeter.h

```diff
@@ -11,7 +11,7 @@ in the source distribution for its full text.
 
 #include "Meter.h"
```

Comment:
Headers not sorted.

Reply:
This does not (or no longer) touch that file. Nothing to care about here, no?